src/js/edu-benefits/5490/config/form.js

```diff
@@ -201,6 +198,9 @@ const formConfig = {
               previousBenefits: _.merge(
                 _.unset('properties.veteranFullName', previousBenefits),
                 {
+                  // Doesn't let us pass even if view:claimedSponsorService is false
+                  //  (which makes the ssn box is hidden)
+                  // required: ['veteranSocialSecurityNumber'],
```

Comment:
This is sort of weird, but probably for the best. The data rjsf sends to the json schema validator has all object fields set to empty objects if they have no data (even though they might get stored in our Redux store as undefined). So the validation will always get triggered. This normally works fine, but our conditional stuff makes it a little complicated.